### PR TITLE
Adds two lines of code to make singularity ships possible.

### PR DESCRIPTION
**Pull request by NiftyRyanyan**
## About The Pull Request

This is my first time using GitHub so don’t be to harsh on me.

But this makes it so uranium floor is immune to the singularity. Currently it’s impossible to build a working singularity on a moving ship because of the open space it creates. So I went with the most simple fix thanks to my lack of knowledge. I’m open of suggestions to fix/ improve this or if I should change the immune floor.

## Why It's Good For The Game
It’s generally good to have the ability to build on of the most iconic engines on your ship

## Changelog
:cl: 
add: uranium floor is now protected from the singularity’s wraith.
/:cl:

### CODE REVIEW

File: code/game/turfs/open/floor/mineral_floor.dm [hunk 1]
```diff
@@ -268,6 +268,9 @@
 			active = 0
 			return
 
+/turf/open/floor/mineral/uranium/singularity_act()
+	return // void crew edit, literally just makes uranium floor immune to the singularity so you now can build one on your ship.
```
Comment (SuperNovaa41) [inline code/game/turfs/open/floor/mineral_floor.dm]:
```suggestion
	return // void crew edit
```

Comment (SuperNovaa41) [reply code/game/turfs/open/floor/mineral_floor.dm]: just this is good enough don't worry

Comment (NiftyRyanyan) [reply code/game/turfs/open/floor/mineral_floor.dm]: There changed it just to voidcrew edit

**Comment by dwasint** [comment]
I like this change

**Comment by SuperNovaa41** [comment]
I do too

**Comment by theta006** [comment]
giga chad

**Pull request merged**